app/jobs/region_cache_warmer_job.rb

```diff
@@ -0,0 +1,33 @@
+class RegionCacheWarmerJob
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :default
```

Comment:
This all seems fine, but this is going to completely saturate the default queue with hundreds of jobs while this runs...would it make more sense to put this in the `low` queue? The cache should still be warmed from the previous run, so these aren't really high or even 'default' priority.

Reply:
The audit logs clog up the low queue in multiples of 100k. Might actually not be so bad to keep this at a slightly higher priority.